pkg/handlers/default.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020-2022 Kinvolk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"os"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/kinvolk/seccompagent/pkg/registry"
+	log "github.com/sirupsen/logrus"
+
+	libseccomp "github.com/seccomp/libseccomp-golang"
```

Comment:
curious: why not collapse these and order alphabetically?

Reply:
I updated this to group external packages together.